infra/gravity/node_commands.go

```diff
@@ -113,6 +113,10 @@ type InstallParam struct {
 	InstallerURL string `json:"installer_url,omitempty"`
 	// OpsAdvertiseAddr is optional Ops Center advertise address to pass to the install command
 	OpsAdvertiseAddr string `json:"ops_advertise_addr,omitempty"`
+	// ServiceUID is an optional param for install's --service-uid flag: https://gravitational.com/gravity/docs/ver/7.x/pack/#service-user
+	ServiceUID uint `json:"service_uid,omitempty"`
```

Comment:
I'm curious if it'd make sense to make this a pointer (`*uint`) to be able to distinguish the case where it's set to a zero vs not set.

Reply:
I'd go with a pointer.

Reply:
Addressed in 786cc87.